src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -5502,180 +5314,141 @@ namespace VariableSpeedCoils {
+            if (varSpeedCoils.MSRatedTotCap(Mode) > varSpeedCoils.MSRatedTotCap(Mode + 1)) {
                 ShowWarningError(state,
                                  format("SizeDXCoil: {} {}, Speed {} Rated Total Cooling Capacity must be less than or equal to Speed {} Rated Total "
                                         "Cooling Capacity.",
-                                        state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).VarSpeedCoilType,
-                                        state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).Name,
+                                        varSpeedCoils.VarSpeedCoilType,
+                                        varSpeedCoils.Name,
                                         Mode,
                                         Mode + 1));
                 ShowContinueError(state,
-                                  format("Instead, {:.2R} > {:.2R}",
-                                         state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedTotCap(Mode),
-                                         state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedTotCap(Mode + 1)));
+                                  format("Instead, {:.2R} > {:.2R}", varSpeedCoils.MSRatedTotCap(Mode), varSpeedCoils.MSRatedTotCap(Mode + 1)));
                 ShowFatalError(state, "Preceding conditions cause termination.");
             }
         }
 
         // convert SHR to rated Bypass factor and effective air side surface area
-        if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).CoolHeatType == "COOLING") {
-            for (Mode = 1; Mode <= state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).NumOfSpeeds; ++Mode) {
-                state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedCBF(Mode) =
-                    CalcCBF(state,
-                            state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).VarSpeedCoilType,
-                            state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).Name,
-                            RatedInletAirTemp,
-                            RatedInletAirHumRat,
-                            state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedTotCap(Mode),
-                            state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedAirVolFlowRate(Mode),
-                            state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedSHR(Mode),
-                            true);
-                if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedCBF(Mode) > 0.0) {
-                    state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSEffectiveAo(Mode) =
-                        -std::log(state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedCBF(Mode)) *
-                        state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedAirMassFlowRate(Mode);
+        if (varSpeedCoils.CoolHeatType == "COOLING") {
```

Comment:
Strike three! You're out!

Reply:
I agree. I also feel that clean up/refactor changes should be on a refactor branch, not in a defect branch where diffs are expected. How do I know if I made a mistake? I think that should be a rule, especially when reviewers can't see the changes unless I can find them again and point them out.

Reply:
I get it. I'm more just pointing this out because PR's are my "callback" for looking at code and engaging with developers who are touching that code. If you feel like something is better left for later, just put a pin in it. I've done that myself in several places if I feel the change is too big to do right now and might hold up the PR, which I do not want to do. But often I don't know.